shared/checks/oval/installed_OS_is_rhv4.xml

```diff
@@ -11,12 +11,9 @@
       <description>The operating system installed on the system is
       Red Hat Virtualization Host 4 or Red Hat Enterprise Host.</description>
     </metadata>
-    <criteria>
-      <extend_definition comment="RHEL7 OS installed" definition_ref="installed_OS_is_rhel7" />
-      <criteria operator="OR">
-        <criterion comment="Red Hat Virtualization Host (RHVH)" test_ref="test_rhvh4_version" />
-        <criterion comment="Red Hat Virtualization Host is based on RHEL (RHELH)" test_ref="test_rhelh4_version" />
-      </criteria>
```

Comment:
@yuumasato should we just remove `test_rhelh4_version` and keep the `installed_OS_is_rhel7`?

Reply:
We can use the `installed_OS_is_rhel7`, but I'd put it in the place of `test_rhelh4_version`.

Reply:
@yuumasato basically we need to handle this:

- RHVH and RHVM in RHV4 DS
- RHELH (aka RHVH) in RHEL7 DS.

Reply:
@redhatrises Something like this?

Reply:
@redhatrises I think we should be fine limiting RHV4 DS to RHVH and RHVM.

But I'd rather not limit RHEL7 DS, and still allow it to evaluate on RHVH due to https://bugzilla.redhat.com/show_bug.cgi?id=1420038.